technic/machines/register/recipes.lua

```diff
@@ -96,7 +96,13 @@ local function register_recipe(typename, data)
 	end
 end
 
+-- Checks for "zzzz_exchangeclone_crafthook" mod so that it won't crash with older versions of ExchangeClone which don't have it.
+local has_exchangeclone = minetest.get_modpath("zzzz_exchangeclone_crafthook")
```

Comment:
iirc the mod name needs to be updated
```suggestion
local has_exchangeclone = minetest.get_modpath("zzzz_exchangeclone_init")
```